src/controllers/slackController/index.ts

```diff
@@ -18,9 +18,11 @@ export default class SlackController {
     const { challenge, event } = slackEventInfo
 
     if (challenge) {
-      return res.send(body)
+      res.send(body)
     }
 
+    res.sendStatus(200)
```

Comment:
Why is this moved from bottom? :P

Reply:
I found out that it is best practice to answer to event or action as quick as possible and then do business logic so that is why I move res.sendStatus(200)

https://api.slack.com/tutorials/intro-to-message-buttons